executor/aggregate.go

```diff
@@ -224,6 +225,13 @@ func (e *HashAggExec) Close() error {
 	for range e.finalOutputCh {
 	}
 	e.executed = false
+
+	// len(e.partialWorkers) == len(e.finalWorkers)
```

Comment:
How cames this assertion?

See https://github.com/pingcap/tidb/blob/5c521c8e43a6706e3f98ed62b7d6544403bb932d/executor/aggregate.go#L254-L255